### PR TITLE
adding wpan utils

### PR DESCRIPTION
**Pull request by jenniexie**
adding wpan utils per Priyanka request

### CODE REVIEW

File: silk/tools/wpan_util.py [hunk 1]
```diff
@@ -101,3 +101,64 @@ def verify_prefix(node_list, prefix, prefix_len=64, stable=True, priority='med',
...
+
+        for entry in childTable:
+            if entry.ext_address == ext_addr:
+                verify(int(entry.rloc16, 16) == int(child.get(wpan.WPAN_THREAD_RLOC16), 16))
+                verify(int(entry.timeout) == int(child.getprop(wpan.WPAN_THREAD_CHILD_TIMEOUT)))
+                mode = int(child.get(wpan.WPAN_THREAD_DEVICE_MODE), 0)
+
+                if i < len(children) - 2:
+
+                    verify(entry.is_ftd() == (mode & wpan.THREAD_MODE_FLAG_FULL_THREAD_DEV != 0))
```
Comment (priyankaCh22) [inline silk/tools/wpan_util.py:138]: Thanks a lot Jennie for this PR.

As as last discussion to have more sed please change the code lines 137 to 141 as below. Thanks again.
                    verify(child.get(wpan.WPAN_NODE_TYPE) == wpan.NODE_TYPE_SLEEPY_END_DEVICE)
                else:
                    verify(entry.is_ftd() == (mode & wpan.THREAD_MODE_FLAG_FULL_THREAD_DEV != 0))
                    verify(entry.is_rx_on_when_idle() == (mode & wpan.THREAD_MODE_FLAG_RX_ON_WHEN_IDLE != 0))

File: silk/node/wpantund_base.py [hunk 2]
```diff
@@ -147,10 +147,16 @@ def form(self, network, role):
...
 
-        command += " -c %s" % network.channel
+        command += " -c {}".format(network.channel)
+
+        if xpanid and not panid:
+            command += " -x {}".format(xpanid)
+        elif panid and not xpanid:
+            command += " -p {}".format(hex(panid))
+        elif xpanid and panid:
+            command += " -x {} -p {}".format(xpanid, hex(panid))
```
Comment (qilinkang) [inline silk/node/wpantund_base.py:158]: Suggest to simplify line 154 to 159 in def form() :
if xpanid:
    command += ....
if panid:
    command += ....

Comment (jenniexie) [reply silk/node/wpantund_base.py:158]: fixed

File: silk/tools/wpan_util.py (hunk 1 above)
Comment (jenniexie) [reply silk/tools/wpan_util.py:138]: done

**Review by priyankaCh22 — APPROVED** [review]
Thanks Jennie. Looks good to me.

**Review by qilinkang — APPROVED** [review]
LGTM

**Review by jwhui — APPROVED** [review]

**Pull request merged**